router/src/main.c

```diff
@@ -73,10 +72,10 @@ static void signal_handler(int signum)
 
 static void check(int fd) {
     if (qd_error_code()) {
-        qd_log(log_source, QD_LOG_CRITICAL, "Router start-up failed: %s", qd_error_message());
-        dprintf(fd, "%s: %s\n", argv0, qd_error_message());
-        close(fd);
-        exit(1);
+    qd_log(QD_LOG_MODULE_ROUTER, QD_LOG_CRITICAL, "Router start-up failed: %s", qd_error_message());
+    dprintf(fd, "%s: %s\n", argv0, qd_error_message());
+    close(fd);
+    exit(1);
```

Comment:
I hit this once before. This is some sort of weird clang-format behavior. It formats this reasonably when asked to reformat whole file, but when formatting only changes, it removes the indent for the body of the if. /me confused